src/Enemy.cpp

```diff
@@ -57,11 +57,47 @@ void Enemy::DisplayHUD() {
 
 
 
-vector<int> Enemy::GetDrops() {
+ItemMap_t Enemy::GetDrops() {
     // Returns the number of items dropped when the enemy is defeated depending on the item won.
-    vector<int> drops;
+    ItemMap_t drops;
+    int ArrowSelector = Common::RandomInt(0, 5);
+    int WhetstoneSelector = Common::RandomInt(0, 1);
+    int PotionSelector = Common::RandomInt(0, 9);
+
     for (int i = 0; i < NUM_ITEMS; i++) {
-        drops.push_back(ReturnItemDrop(i));
+        ITEMTYPE itemType = static_cast<ITEMTYPE>(i + 1);
```

Comment:
We should avoid the for loop and the static_cast, see the code in my [previous comment](https://github.com/tagniam/Turn/pull/135#discussion_r512346171).